CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 ### Changed
+* Make select error description more informative in case
+  of disabled/uninit storage (part of #229).
```

Comment:
```suggestion
  of disabled/uninit storage (#229).
```
should be enough

Reply:
I have read https://github.com/tarantool/crud/pull/297#discussion_r885691328 comment. To be honest, I don't think we should write "part of #XXX" here because now crud CHANGELOG contains only issue/PR numbers.